app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardEditorView.kt

```diff
@@ -46,7 +46,30 @@ class CreditCardEditorView(
         }
 
         save_button.setOnClickListener {
-            saveCreditCard()
+            containerView.hideKeyboard()
+
+            if (state.isEditing) {
+                interactor.onUpdateCreditCard(
+                    guid = state.guid,
+                    creditCardFields = UpdatableCreditCardFields(
```

Comment:
Instead of duplicating creating the same UpdatableCreditCardFields in if/else we can create it once and then use it in different methods.